Modules/_testmultiphase.c

```diff
@@ -952,3 +952,37 @@ PyInit__test_shared_gil_only(void)
 {
     return PyModuleDef_Init(&shared_gil_only_def);
 }
+
+
+#include "datetime.h"
+
+static int
+datetime_capi_client_exec(PyObject *m)
+{
+    _PyDateTimeAPI_Clear();
+    if (PyDateTimeAPI != NULL) {
```

Comment:
I would prefer to avoid macros here:

```suggestion
    if (_PyDateTimeAPI_Get() != NULL) {
```